velox/exec/Task.h

```diff
@@ -992,9 +992,12 @@ class Task : public std::enable_shared_from_this<Task> {
   // Application specific task ID specified at construction time. May not be
   // unique or universally unique.
   const std::string taskId_;
+  // QueryContext should be declared before planFragment as
+  // planFragment may allocate memory in query mempool held by queryContext
```

Comment:
This comment can be removed as it might not be accurate. The plan fragment might not allocate memory from the task or query memory pool. In Prestissimo, it allocate memory from the memory pool in the task or query manager.